README.md

```diff
@@ -1,5 +1,17 @@
 The Apache Drill website is built using [Jekyll](http://jekyllrb.com/).
 
+# Configuring env
+1. Install `ruby`
+2. Install `bundler` and `jekyll` `v2.5.2`:
```

Comment:
If we need specific version, please make an explicit note about that, warning that others versions won't work.

Reply:
Thanks, good point, added this note after this list.